perun-base/src/main/java/cz/metacentrum/perun/controller/model/ServiceState.java

```diff
@@ -90,7 +90,7 @@ public int getTaskId() {
 	 *
 	 * @return Time when was last task scheduled.
 	 */
-	public Date getScheduled() {
+	public LocalDateTime getScheduled() {
```

Comment:
Please change it to internally use ``task.getScheduleAsLong()``. Otherwise you need to implement same supplementary methods as in ``Task`` object, including changes in serializer.
Same goes for ``getStartTime`` and ``getEndTime()``.